forge/ee/lib/billing/index.js

```diff
@@ -546,7 +546,7 @@ module.exports.init = async function (app) {
                             // needed by closeSubscription. This is to ensure when the
                             // stripe callback arrives we don't trigger a suspension of
                             // the team resources.
-                            await stripe.subscriptions.del(subscription.subscription, {
+                            await stripe.subscriptions.del(existingSubscription, {
```

Comment:
Easy to miss, I wouldn't expect `subscription.subscription` to be `''` but indeed I see it on line 528 above: https://github.com/FlowFuse/flowfuse/blob/9df9a6f4d37452f5322acda65a3a8082e83452a3/forge/ee/lib/billing/index.js#L528